.github/workflows/notify-non-core-qn.yml

```
@@ -1,37 +1,39 @@
-      ALLOWED_USERNAMES: ${{ vars.ALLOWED_USERNAMES }}
-      QUESTION_KEYWORDS: ${{ vars.QUESTION_KEYWORDS }}
-      QUESTION_LABELS: ${{ vars.QUESTION_LABELS }}
-      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+      ALLOWED_USERNAMES: ${{ vars.ALLOWED_USERNAMES || '' }}
+      QUESTION_KEYWORDS: ${{ vars.QUESTION_KEYWORDS || '' }}
+      QUESTION_LABELS: ${{ vars.QUESTION_LABELS || '' }}
+      SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK || '' }}
+
     steps:
       - name: Check and Notify
         uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)